app/i18n/cz/conf.php

```diff
@@ -136,31 +136,31 @@
 	'reading' => array(
 		'_' => 'Čtení',
 		'after_onread' => 'Po „označit vše jako přečtené“',
-		'always_show_favorites' => 'Vy výchozím nastavení zobrazit všechny články v oblíbených',
+		'always_show_favorites' => 'Ve výchozím nastavení zobrazit všechny články v oblíbených',
 		'article' => array(
 			'authors_date' => array(
-				'_' => 'Authors and date',	// TODO
-				'both' => 'In header and footer',	// TODO
-				'footer' => 'In footer',	// TODO
-				'header' => 'In header',	// TODO
-				'none' => 'None',	// TODO
+				'_' => 'Autoři a datum',
+				'both' => 'V záhlaví a zápatí',
+				'footer' => 'V zápatí',
+				'header' => 'V záhlaví',
+				'none' => 'Žádný',
 			),
 			'feed_name' => array(
-				'above_title' => 'Above title/tags',	// TODO
-				'none' => 'None',	// TODO
-				'with_authors' => 'In authors and date row',	// TODO
+				'above_title' => 'Nad názvem/štítky',
+				'none' => 'Žádný',
+				'with_authors' => 'V řádku s autory a datem',
 			),
-			'feed_title' => 'Feed title',	// TODO
+			'feed_title' => 'Název feedu',
 			'tags' => array(
-				'_' => 'Tags',	// TODO
-				'both' => 'In header and footer',	// TODO
-				'footer' => 'In footer',	// TODO
-				'header' => 'In header',	// TODO
-				'none' => 'None',	// TODO
+				'_' => 'Štítky',
+				'both' => 'V záhlaví a zápatí',
+				'footer' => 'V zápatí',
+				'header' => 'V záhlaví',
+				'none' => 'Žádné',
 			),
 			'tags_max' => array(
-				'_' => 'Max number of tags shown',	// TODO
-				'help' => '0 means: show all tags and do not collapse them',	// TODO
+				'_' => 'Max number of tags shown',
+				'help' => '0 means: show all tags and do not collapse them',
```

Comment:
it is still an TODO

Reply:
Fixed this and also found another missing in `sub.php` so i updated this one aswell.